release-notes.md

```diff
@@ -20,6 +20,18 @@ To enable this to all partners run:
 ```
 php deployment/updates/scripts/2023_01_03_enable_studio_V3_permission_to_all_partners.php
 ```
+
+## Configure time frame in hours to look for recent Webex transcripts ##
```

Comment:
just write it as Webex transcripts support in the headline and in the config section explain the purpose of this field in the config